source/deployment/sso-saml-onelogin.rst

```diff
@@ -85,12 +101,9 @@ Create a OneLogin Connection App for Mattermost SSO
 Configure SAML Sign-in for Mattermost
 --------------------------------------
 
-1. Start the Mattermost server, then sign in to Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**.
-  a. In the **SAML SSO URL** field, paste the value for the OneLogin **SAML 2.0 Endpoint (HTTP)** that you copied earlier.
-  b. In the **Identity Provider Issuer URL** field, paste the value for the OneLogin **Issuer URL** that you copied earlier.
-  c. In the **Identity Provider Public Certificate** field, upload the OneLogin X.509 PEM certificate file that you downloaded earlier.
-
-  .. image:: ../../source/images/okta_10_mattermost_basics.PNG
+1. Start the Mattermost server, then sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**.
+  a. Enter the **OneLogin Issuer URL** into the **Identity Provider Metadata URL** field.
+  b. Select **Get SAML Metadata from IdP** to download the metadata.
 
 2. Configure Mattermost to verify the signature.
```

Comment:
@icelander - Your documentation feedback included: "Add Service Provider Identifier configuration here:" for this step. Can you provide more context for what's required, please?

Reply:
This looks good